PronounDB/index.js

```diff
@@ -179,7 +179,7 @@ export default class PronounDB extends BasePlugin {
         }
 
         const patched = new WeakSet();
-        const REGEX = /user.*contextmenu/i;
+        const REGEX = /useUserRolesItems/i;

```

Comment:
This will only work for the `GuildChannelUserContextMenu`, and not for `DMUserContextMenu` etc. This needs proper fixing as I did in my Copier plugin.